### DISCUSSION TITLE
Completely Delete Plug in and re-setup

### DISCUSSION THREAD

**Discussion by thrangster** [homebridge-simplisafe3/homebridge-simplisafe3#385] [Q&A]
So suddenly, the Simplisafe plug in stopped working with HomeKit  - the devices were there, but all "No Response" - I tried to reauthenticare within the app, and then things fell further apart... I cannot get to the point where I see the com.mobile... url in Chrome or Firefox to paste back in, and it seems to be connected to new/additional steps to Verify my Identity on the Simplisafe site... either I cannot submit the six digit code I've received, or if I can, I am then presented with another screen asking me to Login in faster...

![image](https://user-images.githubusercontent.com/68309317/226415959-7088ce91-4813-4aa1-b948-c62896288c69.png)

Clicking No Thanks or Remind Me later doesn't do anything - clicking on Login In Faster brings up a window to Create a Passkey, and either proceeding or cancelling leads me to an error window...

![image](https://user-images.githubusercontent.com/68309317/226416306-9ad10a06-bf43-4064-8bf6-fa9559cfad22.png)

It seems to be looking to passkey resources on a mobile device, and it is getting quite confused because I am on a desktop browser.

Additional, I delete the plug (induing the option to delete the config) to reinstall to see if the URL it was creating was the issue. When I reinstall, the plug in indicate the Authentication is already setup, which it is not (and the service is not running).

So two things - Is there a command line to deep delete the simplisafe directory?

And 2, is there a way to get the authentication to work again or am I doing something wrong? I search every tab on the java console and could not find the com.mobile response url (or whatever its called)

Thanks

**Comment by shamoon** [comment]
Hmm, I can't recreate the issue here, see screenshots. Thats in Chromium and using an account with email 2FA and sms 2FA. I tried a third time with Google Chrome and it offered me a similar option as you noted above, I chose "No Thanks" and it showed the URL in the error console.

You can delete the file HOMEBRIDGE_DIR/simplisafe3auth.json

<img width="1520" alt="Screenshot 2023-03-20 at 4 19 54 PM" src="https://user-images.githubusercontent.com/4887959/226488131-6611c2e4-0971-4f07-8304-4e758c469c1b.png">
<img width="1528" alt="Screenshot 2023-03-20 at 4 21 15 PM" src="https://user-images.githubusercontent.com/4887959/226488132-32195bd8-16e2-4524-8b7b-6447ebc84b59.png">
<img width="1395" alt="Screenshot 2023-03-20 at 4 34 07 PM" src="https://user-images.githubusercontent.com/4887959/226488134-08319a65-7c0f-43cb-afb0-10f45b78e707.png">

**Comment by jsmi1462** [comment]
Hi, thanks for your great work on this plugin. Just wanted to chime in and say I am having the exact same issue. When I click no thanks or later, nothing happens. Clicking on verify creates the same issue OP is having, with the error splash screen. In the console, nothing at all is displayed on the error page. I have used Edge, Chrome, and Firefox.

**Reply by shamoon** [reply]
As noted above, try deleting the auth file, deleting browser cache etc or authenticating in private mode. I know for sure Chrome/Chromium shows the URL you need in the console, havent checked the others in a while. See above screenshots.

**Reply by jsmi1462** [reply]
Thanks for your warp speed reply. Got it. I have never used JavaScript, but I am assuming that I can do that from settings-JSON config-delete HOMEBRIDGE_DIR/simplisafe3auth.json? I am not at all familiar with Java syntax and every result I have googled on deleting files in Java have been way above my pay grade. I must be doing something wrong because it's turning up a config error.

**Reply by shamoon** [reply]
No sorry, nothing to do with the config json, thats an actual file you'll need to find (location varies slightly by install type) and delete the file `simplisafe3auth.json`

**Reply by jsmi1462** [reply]
Understood. Thanks!

**Reply by jsmi1462** [reply]
I was able to fix my instance of the plugin after I reimaged HomeBridge onto my RPi. I was never able to get to the 2FA screen, but I could copy the link from the Login Faster page, just for anyone else looking. You do not have to get past that screen. I also never got an email or text that I had to approve. Thanks again for a great plugin that I use all the time!
